banking/src/test/java/rocks/kata/AppTest.java

```diff
@@ -0,0 +1,115 @@
+package rocks.kata;
+
+import org.junit.jupiter.api.Test;
+
+import rocks.kata.DatabaseClasses.*;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.function.Executable;
+
+class AppTest {
+
+    DateProvider dateProvider = new DateProvider("d.M.yyyy");
+    AccountOperationDatabaseFormatter formatter = new AccountOperationDatabaseFormatter();
+    AccountOperationsDatabase database = new AccountOperationsDatabase(formatter);
+
+    @Test
+    void newlyCreatedAccountShouldResultInCorrectDatabasePrintedStatement() {
+        var account = new Account(database);
+        String currentDate = dateProvider.getCurrentDate();
+
+        String statement = account.printStatement();
+
+        assertEquals("Date\t\tAmount\tBalance\n" + currentDate + "\t0\t0", statement);
```

Comment:
to chyba nie jest zgodne z wymaganiami - nigdzie w przykładzie nie ma takiego wpisu

Reply:
Yup